docs/develop/power-automate-parameters-returns.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Script parameter and return types in Power Automate
+description: How and why to pass data to and from Office Scripts with Power Automate.
+ms.date: 12/19/2022
+ms.localizationpriority: medium
+---
+
+# Script parameter and return types in Power Automate
+
+Power Automate chains together separate programs into a single automated workflow. Each connector has different parameters it accepts and different values it returns. Your scripts can be written to expand the Run script Power Automate action to get additional input or give output.
```

Comment:
For your consideration — It took me a couple reads to understand this sentence. I think adding quotes to indicate that run and script are one item ("Run script") would make the sentence easier to parse. 
```suggestion
Power Automate chains together separate programs into a single automated workflow. Each connector has different parameters it accepts and different values it returns. Your scripts can be written to expand the "Run script" Power Automate action to get additional input or give output.
```